pkg/apiOracle/valueOracle.go

```diff
@@ -76,18 +78,22 @@ func writeOutHistory() {
 	psrSavedDataTmp := psrSavedData + ".tmp"
 	err = ioutil.WriteFile(psrSavedDataTmp, data, 0644)
 	if err != nil {
-		logger.Error("failed to write out PSR values to %s: %s", psrSavedDataTmp, err.Error())
+		level.Error(logger).Log(
+			"msg", "failed to write out PSR values",
+			"psrSavedDataTmp", psrSavedDataTmp,
+			"err", err.Error(),
+		)
 		return
 	}
 	// Rename tmp file to old file (should be atomic on most modern OS)
 	err = os.Rename(psrSavedDataTmp, psrSavedData)
 	if err != nil {
-		logger.Error("failed move new PSR save onto old: %s", err.Error())
+		level.Error(logger).Log("msg", "failed move new PSR save onto old", "err", err.Error())
```

Comment:
```suggestion
		level.Error(logger).Log("msg", "move new PSR save onto old", "err", err)
```

We just describe the action see the coding guidelines. No log msg should include the words fail, error, problem etc.